Config/Config.hs

```diff
@@ -4,6 +4,10 @@ import IHP.Prelude
 import IHP.Environment
 import IHP.FrameworkConfig
 
-instance FrameworkConfig where 
+
+config :: IO FrameworkConfig
+config = let
```

Comment:
This way it's not very clear how to e.g. override attributes

E.g. having it this way it's more clear:


```haskell
config = do
    let environment = Development
    let appHostName = "localhost"

    config <- defaultFrameworkConfig environment appHostName

    config
        |> set #mailServer SES { accessKey = "..", secretKey = "..", region = "eu-west-1" }
        |> modify #sessionCookie (\cookie -> cookie {
            setCookieMaxAge = Just (fromIntegral (60 * 60 * 24 * 90))
        })

Reply:
Agreed! cool syntax!

Reply:
Expanding defaultFrameworkConfig into developer land to solve the late binding issue also fixed this